### ISSUE TITLE
Upgrade rxjs dependency to 6.x?

### ISSUE THREAD

**Issue by manoldonev** [Kinvey/nativescript-sdk#34]
Hey guys, when are you planning to upgrade your rxjs dependency to latest (rxjs6 is official now)?

Currently kinvey-nativescript-sdk references rxjs@5.5.x -- that will cause a problem with the imminent release of Angular6 support in NativeScript (ng6 implies rxjs@6.x and it comes with its fair share of breaking changes).

So the end result is that if you create a {N} app with Angular6 and kinvey-nativescript-sdk it cannot even compile unless one installs the compatibility layer provided by [rxjs-compat](https://github.com/ReactiveX/rxjs/blob/master/MIGRATION.md) and we would eventually like to drop it to reduce app size.

**Comment by manoldonev** [comment]
@thomasconner Can you check this?

**Comment by thomasconner** [comment]
Yes I can

**Comment by docHologram** [comment]
Since `kinvey-nativescript-sdk/package.json` still lists rxjs@5.5.x as a dependency, a quick workaround is to change lines 1-3 in `kinvey.d.ts` from:

```
import {
  Observable
} from 'rxjs/Observable';
```
to:
```
import {
  Observable
} from 'rxjs';
```

**Comment by manoldonev** [comment]
@thomasconner it has been 2.5 months now that this issue is open. Any updates?

**Comment by thomasconner** [comment]
I have just submitted a PR that updates rxjs to version 6. Our next release is scheduled for August 8. It will be part of that release.

**Comment by manoldonev** [comment]
@thomasconner any updated estimates for the next release?

**Comment by thomasconner** [comment]
The release will be made by the end of the day next Wednesday (August 22).

**Comment by manoldonev** [comment]
@thomasconner any updates on this one? it has been almost a month after August 22 but no release?

**Comment by thomasconner** [comment]
@manoldonev We are in the process of making a release within the next week that will contain this fix.

**Comment by thomasconner** [comment]
This has been fixed with [3.11.7](https://www.npmjs.com/package/kinvey-nativescript-sdk/v/3.11.7)
